pkg/cli/alpha/config-gen/cert-generation-filter.go

```diff
@@ -107,6 +107,13 @@ func (c CertFilter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error) {
 		if err != nil {
 			return nil, err
 		}
+		// TODO: verify if this works, since `yaml.Append` requires a node to be created for clientConfig
```

Comment:
have added a comment here. `SequenceNode` seems to be apt, but not sure if this works as required as there are no tests for this yet.

Reply:
SequenceNode is the correct type. Does the list get added as expected when you test this locally?